### ISSUE TITLE
Computed properties can not be observed, if they have dots in keypath

### ISSUE THREAD

**Issue by Minstel** [ractivejs/ractive#2807]
## Description:
Seems like computed properties can not be observed, if they have dots in keypath. No warning or error arises.
## Versions affected:
0.8.7
## Platforms affected:
Chrome 55.0.2883.87 m
## Reproduction:
Just create a computed property with dot(s) in keypath, and then try to observe it's value.
https://jsfiddle.net/Minstel/4v7fysj0/
## Code:

```
var r = new Ractive({
	data: {
    	zoo: 4 + 5,
	},
	computed: {  	
    	'bar\\.foo': '4 + ${zoo}',
	}
});

r.observe('zoo', function(newValue, oldValue, key) {
	console.log('changed zoo: ', newValue, oldValue, key);
});

// Try to observe computed with dots
r.observe('bar.foo bar\.foo bar\\.foo bar\\\.foo bar\\\\.foo', function(newValue, oldValue, key) {
	console.log('changed computed: ', newValue, oldValue, key);
});

r.set('zoo', 10);
```

**Comment by ghost** [comment]
To clarify, the change can be observed (the handler executes), but the old and new values aren't provided. Very minimum reproduction: https://jsfiddle.net/4mot9kqL/

```
var r = new Ractive({
  data: {
    zoo: 4 + 5,
  },
  computed: {  	
    'bar.foo': '4 + ${zoo}',
  }
});

r.observe('bar.foo', function(newValue, oldValue, key) {
  console.log('changed computed: ', newValue, oldValue, key);
});

r.set('zoo', 10);
```

It also used to work in 0.7.3: https://jsfiddle.net/jfyw05d8/

**Comment by Minstel** [comment]
@fskreuz it's initial call to observe, that you see. Non-initial observe, when value is recomputed, is not performed. Try to set `{init: false}` in options. 
I've changed my issue fiddle a little, so that it was clear.

Also when using computed keypath without escaping dot, like you provide, value can not be obtained with `get` function. According to Ractive changelog, as of 0.8.0 we should escape dots.

It's also the other face of bug. Non-computed properties, if dot in keypath is not escaped, are presented as object ierarchy, and we can get their values with `get`. With computed properties, we can only get value if dot was escaped when setting value.

**Comment by evs-chris** [comment]
You should still define computeds with plain dotted keypaths e.g. `computed: { 'foo.bar': '...' }` as you would with any other data. You should access them with escaped keypaths e.g. `r.get('foo\\.bar')` as you would with any other data.

This is now fixed in 0.8-dev and edge, and will be published when 0.8.8 is released. Thanks for the report!
